<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMakeSelf</class>
 <widget class="QMainWindow" name="QMakeSelf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>509</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>509</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>509</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QMakeSelf</string>
  </property>
  <widget class="QWidget" name="wCentral">
   <property name="minimumSize">
    <size>
     <width>920</width>
     <height>445</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>920</width>
     <height>451</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
    <item>
     <widget class="QGroupBox" name="gbArhiveRoot">
      <property name="title">
       <string>Корень архива</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTreeView" name="tvArchiveRoot"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hl_params">
      <item>
       <layout class="QVBoxLayout" name="vl_main_params">
        <item>
         <widget class="QGroupBox" name="gbCompression">
          <property name="title">
           <string>Сжатие</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="hl_comp" stretch="9,1">
             <item>
              <widget class="QComboBox" name="cbCompressionType"/>
             </item>
             <item>
              <widget class="QSpinBox" name="sbCompressionRatio">
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::UpDownArrows</enum>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>9</number>
               </property>
               <property name="value">
                <number>9</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="leTarAdditional">
             <property name="placeholderText">
              <string>Дополнительные параметры tar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbSafeControl">
             <property name="title">
              <string>Контрольные суммы</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QCheckBox" name="cbCRC">
                <property name="text">
                 <string>CRC</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbMD5">
                <property name="text">
                 <string>MD5</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbUnpack">
          <property name="title">
           <string>Распаковка</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="leLabel">
             <property name="placeholderText">
              <string>Заголовок программы</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbUnpackType"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hl_extract_path" stretch="9,1">
             <item>
              <widget class="QLineEdit" name="leDestination">
               <property name="placeholderText">
                <string>Путь назначения</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbSelectPath">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/pic/add.png</normaloff>:/pic/add.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="cbCopyToTemporary">
             <property name="text">
              <string>Скопировать установщик во временную папку</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gbAdditional">
        <property name="title">
         <string>Дополнительно</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="cbFollowSymLinks">
           <property name="text">
            <string>Следовать по ссылкам в корне архива</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbUnpackProgress">
           <property name="text">
            <string>Не показывать прогресс распаковки</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbXtermAuto">
           <property name="text">
            <string>Запретить автозапуск xterm</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbXtermClose">
           <property name="text">
            <string>Закрывать xterm после распаковки</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hl_initscript_path" stretch="9,1">
           <item>
            <widget class="QLineEdit" name="leDestinationInitScript">
             <property name="placeholderText">
              <string>Путь к пост-сценарию</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbSelectPathInitScript">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/pic/add.png</normaloff>:/pic/add.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hl_license_path" stretch="9,1">
           <item>
            <widget class="QLineEdit" name="leDestinationLicense">
             <property name="placeholderText">
              <string>Путь к файлу лицензии</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbSelectPathLicense">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/pic/add.png</normaloff>:/pic/add.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hl_lsm_path" stretch="9,1">
           <item>
            <widget class="QLineEdit" name="leDestinationLSM">
             <property name="placeholderText">
              <string>Путь к LSM файлу</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbSelectPathLSM">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/pic/add.png</normaloff>:/pic/add.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hl_header_path" stretch="9,1">
           <item>
            <widget class="QLineEdit" name="leDestinationHeader">
             <property name="placeholderText">
              <string>Путь к header-сценарию</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbSelectPathHeader">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/pic/add.png</normaloff>:/pic/add.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mbMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="tbToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="spStatusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
